### DISCUSSION TITLE
"No working indexers available"

### DISCUSSION THREAD

**Discussion by LazerDickMcCheese** [cross-seed/cross-seed#789] [Q&A]
Fresh install from a Docker YAML, I added my URLs and Prowlarr APIs. Here's the log:

2024-09-09 20:32:34 [32minfo[39m: cross-seed v5.9.2
2024-09-09 20:32:34 [36mverbose[39m: [configdump] {
  torznab: [],
  matchMode: 'safe',
  dataCategory: 'cross-seed-data',
  linkType: 'symlink',
  skipRecheck: false,
  maxDataDepth: 2,
  torrentDir: '/torrents',
  outputDir: '/cross-seeds',
  includeNonVideos: false,
  includeEpisodes: false,
  includeSingleEpisodes: false,
  fuzzySizeThreshold: 0.02,
  verbose: false,
  action: 'save',
  duplicateCategories: false,
  delay: 10,
  snatchTimeout: 30000,
  searchTimeout: 30000,
  searchLimit: 0,
  port: 2468,
  apiAuth: true
}
2024-09-09 20:32:34 [32minfo[39m: Validating your configuration...
2024-09-09 20:32:34 [31merror[39m: no working indexers available

Every time I start the container, cross-seed deletes the torznab URLs then gives the error above.

**Comment by mmgoodnow** [comment]
Your config dir is likely not mapped correctly. Cross-seed does not rewrite the config file

**Reply by LazerDickMcCheese** [reply]
If I start the container, it shuts itself off, and the torznab section is now empty. If I add the URLs again and restart the container, the URLs disappear.

I just checked the configs, they are mapped correctly and the container is writing to the directory.

**Reply by bakerboy448** [reply]
Then  your edits are not saving

No functionality exists for xseed to nuke the Torznab settings as you claim
